GeosCore/drydep_mod.F90

```diff
@@ -1709,6 +1718,34 @@ SUBROUTINE DEPVEL( Input_Opt, State_Chm, State_Diag, State_Grid, &
                 ! Particle diameter, convert [m] -> [um]
                 DIAM  = A_RADI(K) * 2.e+0_f8
 
+                IF ((SpcInfo%Name .EQ. 'DST1') .OR.      &
+                    (SpcInfo%Name .EQ. 'DSTAL1') .OR.    &
+                    (SpcInfo%Name .EQ. 'NITD1') .OR.     &
+                    (SpcInfo%Name .EQ. 'SO4D1')) THEN
+                    DIAM = 0.66895E-6
+                ENDIF
```

Comment:
Replace these string comparisons with comparisons to integer flags returned by the `Ind_` function.  String comparisons are much slower than integer comparisons.

Reply:
Will do. Appreciate it if there can be a example code snippet for using integer flags in GEOS-Chem. Thanks!